run_scripts/1_coda_get.sh

```diff
@@ -15,7 +15,7 @@ DATA_ROOT=$3
 ./checkout_coda_v2.sh "$CODA_V2_ROOT"
 
 cd "$CODA_V2_ROOT/data_tools"
-git checkout "9a9a8e708e3f20f37848a6b02f79bcee43e5be3b"  # (master which supports segmenting)
+git checkout "94a55d9218fb072ef2c15ee2c27c4214b036bd2f"  # (master which supports LastUpdated)
```

Comment:
Just to check if this branched from master which supports segmenting?

Reply:
This commit is still on master (HEAD at time of comment), so it does segmenting + LastUpdated. Because segmenting has been supported by all running pipelines for many months now I was taking it as given that the pipeline had that. I just wanted a quick way to be able to check which coda features each pipeline has as we do the next round of upgrades, which is hard since we're not importing a versioned library, hence the comment